locale/en/knowledge/HTTP/servers/how-to-create-a-HTTPS-server.md

```diff
@@ -7,11 +7,6 @@ difficulty: 1
 layout: knowledge-post.hbs
 ---
 
```

Comment:
locale/en/knowledge/HTTP/servers/how-to-create-a-HTTPS-server.md
[SquirrelSetup.log](https://github.com/nodejs/nodejs.org/files/6244157/SquirrelSetup.log)

Reply:
@@ -1,16 +1,12 @@